handsontable.d.ts

```diff
@@ -97,6 +99,7 @@ declare namespace _Handsontable {
     rootDocument: Document;
     rootElement: HTMLElement;
     rootWindow: Window;
+    rowIndexMapper: Handsontable.RecordTranslation.IndexMapper;
```

Comment:
Can I ask you to add TS tests where you can prove that this definition works? Something like (https://github.com/handsontable/handsontable/blob/master/test/types/methods.types.ts#L6):
```js
hot.rowIndexMapper.executeBatchOperations();
hot.rowIndexMapper.registerMap();
hot.rowIndexMapper.unregisterMap();
// ... more
```

The same should be done for `hot. executeBatchOperations()` method and `hot.columnIndexMapper` prop.